### PR TITLE
Adding Company Data Health Checks Documentation

### PR DESCRIPTION
**Pull request by lior-statsig**
Adding Company Data Health Checks Documentation

Adding company data health checks documentation

### CODE REVIEW

**Comment by jasonwzm** [comment]
🚀 Deployed on https://649b2e7222da740084d06cd1--cozy-fox-0defec.netlify.app

File: docs/metrics/health-checks.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+To monitor the health of your data,
+
+- Navigate to **Metrics** in the left-hand navigation panel in the Statsig console
+- Select the **events** tab at the top
+
+- **Data Health Checks Card**
+
+  ![Screen Shot 1402-04-06 at 15 31 53](https://github.com/statsig-io/docs/assets/133702265/07997263-9840-4e09-ae2f-e709e852df71)
+
+  - The data health checks card displays the results of the health checks that are run on your data. If the data passes all the health checks, the card is collapsed by default. If any health check fails, the card is automatically expanded revealing the health check results.
```
Comment (tvh-statsig) [inline docs/metrics/health-checks.md:18]: delete: If the data passes all the health checks, the card is collapsed by default.

File: docs/metrics/health-checks.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  - Hover over the icon and click on a check for more details.
+
+- Viewing these health checks can help you identify any anomalies or issues with the experiment setup. By monitoring these health checks, you can ensure the reliability and accuracy of your experiment results. To monitor the health of your Data, the following health checks are performed.
```
Comment (tvh-statsig) [inline docs/metrics/health-checks.md:21]: "experiment setup" -> data

"accuracy of your experiment results" -> accuracy of your metrics

**Review by tvh-statsig — APPROVED** [review]
nicenice

**Pull request merged**